devicetypes/smartthings/zwave-metering-dimmer.src/zwave-metering-dimmer.groovy

```diff
@@ -229,6 +263,14 @@ def setLevel(level, rate = null) {
 
 def configure() {
 	log.debug "configure()"
+
+	if (isAeotecNanoDimmer()) {
+		state.configured = false
+		state.minDimmingLevel = false
+		state.maxDimmingLevel = false
+		response(getAeotecNanoDimmerConfigurationCommands())
```

Comment:
you should just add these commands to `result` I think?

Reply:
right! fixed